patches/cli-window-open.diff

```diff
@@ -8,7 +8,7 @@ To test:
 2. Open code-server
 3. Open terminal
 4. Open another code-server window
-5. Run code-server with a file or directory argument
+5. Run node ./out/node/entry.js with a file or directory argument
```

Comment:
FYI `node .` can be used as a shortcut since it will use the `main` entry in `package.json` which points to `./out/node/entry.js`.

Reply:
TIL!